src/groups/special_euclidean.jl

```diff
@@ -494,6 +549,128 @@ function exp_lie!(G::SpecialEuclidean{TypeParameter{Tuple{3}}}, q, X)
     return q
 end
 
+@doc raw"""
+    jacobian_exp_inv_argument(
+        M::SpecialEuclidean{TypeParameter{Tuple{2}}},
+        p,
+        X,
+    )
+
+Compute Jacobian matrix of the invariant exponential map on [`SpecialEuclidean`](@ref)`(2)`.
+The formula reads
+````math
+\begin{pmatrix}
```

Comment:
one could maybe do the fractions as ``\frac{1}{\theta}(\sin(\theta)`` ? 

And there are a few `\theta`s that could be turned into their UTF8s like the other are.

Reply:
Something like in the commit I've just made?

Reply:
Perfect.